src/ghe_api.py

```diff
@@ -31,7 +31,7 @@ def get_access_token(code):
 
 def get_login(access_token):
 	if DEV_MODE:
-		return DEV_MODE_LOGIN
+		return access_token
```

Comment:
This method is supposed to return a username apparently. If you look at `sample_config.py`, it is defined with @shreyas208's username. In the real world, we use the access token to get the login (user).